src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {

```

Comment:
check common mistakes 
```suggestion
public interface Drawable {

```